### DISCUSSION TITLE
Is there a plan for a newer openfoam?

### DISCUSSION THREAD

**Discussion by bladesaber** [nextfoam/baram#76] [General]
Hi, thanks for all the work. Openfoam version 6 seems a bit old. Is there a plan for a newer openfoam eg. version 11?

**Comment by kdg5424** [comment]
Hi @bladesaber . This is Donggyu.

BARAM v24 uses NextFOAM v24 solver. And NextFOAM v24 is based on OpenFOAM v2312 (ESI).

BARAM v6 used OpenFOAM v6. I think you confused with BARAM v6.
We're not supporting BARAM v6 now.
Do you have any other question about BARAM?

Best regards

Donggyu kim

**Comment by bladesaber** [comment]
Oh. I believe I was confused by the version control of openfoam. It seems like openfoam.org and openfoam.com are two different projects, and the solvers from both projects aren't compatible. But I still can not find information about buoyantSimpleNFoam, where I can find the usage of this solver?

Best regards and thanks for all your work.

**Comment by kdg5424** [comment]
buoyantSimpleNFoam is the buoyantSimpleFoam solver that we fixed.
We improved the stability of the solver.
And you could find on the page. -Our solver github page.0
[NextFOAM page](https://github.com/nextfoam/nextfoam-cfd)

We suggest you to why don't use our solver.

Thanks

Regards
Donggyu
